pkgs/development/libraries/postgis/default.nix

```diff
@@ -89,6 +89,10 @@ in rec {
     sha256 = "02gsi1cm63kf0r7881444lrkzdjqhhpz9a5zav3al0q24nq01r8g";
     sql_srcs = ["postgis.sql" "spatial_ref_sys.sql"];
     builtInputs = [gdal json_c pkgconfig];
+
+    # postgis config directory assumes /include /lib from the same root for json-c library
+    NIX_LDFLAGS="-L${json_c.out}/lib";
```

Comment:
@joachifm  I couldn't find getLib but I just changed it to use mkLibraryPath. Hopefully that has a similar effect?

Thank you for pointing this out.